shell/plugins/dashboard-store/actions.js

```diff
@@ -402,7 +403,7 @@ export default {
   //  filter: Filter by fields, e.g. {field: value, anotherField: anotherValue} (default: none)
   //  limit: Number of records to return per page (default: 1000)
   //  sortBy: Sort by field
-  //  sortOrder: asc or desc
+  //  sortOrder: asc or desc // TODO: RC when is this ever set (even in findAll), and here?
```

Comment:
it's not but I do believe it is supported by steve (although I think it may have changed recently with the pagination changes they're doing). The worker will support sort direction by prefixing descending sort columns with a "-". So passing a sortBy array of fields like ["namespace", "-restarts"] will sort by namespace in ascending order first and then restarts in descending order second which should be inline with the more recent changes: https://github.com/rancher/steve#sort

Reply:
i wasn't sure if it was actually wired in anywhere, don't think the sortable table passes it through?

Reply:
Have moved the comment to findAll, which is must have been for before.

Will test the plumbing in of sortOrder in PR4